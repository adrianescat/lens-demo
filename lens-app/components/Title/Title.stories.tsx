import { Story } from '@storybook/react';
import Title, { Props } from './Title';

export default {
  title: 'Title',
  component: Title,
};

const Template: Story<Props> = (args) => <Title {...args} />;

export const Primary = Template.bind({});
Primary.args = {};