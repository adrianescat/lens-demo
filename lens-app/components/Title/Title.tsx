import React, { ReactElement } from 'react';
import classNames from 'classnames';
import styles from './Title.module.scss'

export interface Props {
  className?: string;
}

export default function Title(props: Props): ReactElement {
  return (
    <h1 className={classNames(styles.title, props.className)}>
      LensApp
    </h1>
  )
}