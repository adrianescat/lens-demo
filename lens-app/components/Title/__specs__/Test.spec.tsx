import React from 'react';
import { shallow } from 'enzyme';
import Title, { Props } from '../Title';

let wrapper: any;
let props: Props;

beforeEach(() => {
  props = {};
  wrapper = shallow(<Title {...props} />);
});

test('should match snapshots', () => {
  expect(wrapper).toMatchSnapshot();
});