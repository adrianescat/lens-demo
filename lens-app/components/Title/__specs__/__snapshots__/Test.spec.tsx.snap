// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshots 1`] = `
<h1
  className="title"
>
  LensApp
</h1>
`;
